ports/esp32/esp32_rmt.c

```diff
@@ -121,14 +137,8 @@ STATIC mp_obj_t esp32_rmt_make_new(const mp_obj_type_t *type, size_t n_args, siz
 STATIC void esp32_rmt_print(const mp_print_t *print, mp_obj_t self_in, mp_print_kind_t kind) {
     esp32_rmt_obj_t *self = MP_OBJ_TO_PTR(self_in);
     if (self->pin != -1) {
-        mp_printf(print, "RMT(channel=%u, pin=%u, source_freq=%u, clock_div=%u",
+        mp_printf(print, "RMT(channel=%u, pin=%u, source_freq=%u, clock_div=%u)",
```

Comment:
Should we display the idle level and, if configured, the carrier details?

Reply:
Yeah, I just copied in your simplification here. I wasn't sure if I should extend this to be complete or just save the bytes. Away for a few days, but I can fill this out when I get back if it seems like a good idea.